aws/v19.1.1/announcement.md

```diff
@@ -0,0 +1 @@
+**Workload cluster release v19.1.1 for AWS is available**. This is a patch release that provides new `security-bundle` version consisting of two new components in preparation for PSS migration. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v19.1.1/).
```

Comment:
Nitpick just because there's stuff already in the bundle besides the new PSS migration stuff.
```suggestion
**Workload cluster release v19.1.1 for AWS is available**. This is a patch release that provides a new `security-bundle` version introducing new components in preparation for PSS migration. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v19.1.1/).
```